packages/requisitions/src/RequestRequisition/DetailView/Toolbar/ToolbarActions.tsx

```diff
@@ -1,49 +1,106 @@
 import React from 'react';
 import {
+  Switch,
+  Grid,
   Autocomplete,
   InputWithLabelRow,
   useTranslation,
   useConfirmationModal,
 } from '@openmsupply-client/common';
-import { useRequestFields, useIsRequestDisabled } from '../../api';
+import {
+  useRequestFields,
+  useIsRequestDisabled,
+  useHideOverStocked,
+} from '../../api';
 
 const months = [1, 2, 3, 4, 5, 6];
 
 export const ToolbarActions = () => {
+  const { on, toggle } = useHideOverStocked();
```

Comment:
used zustand because I need this toggle to be the same as the one used in the hook. 

related https://github.com/openmsupply/openmsupply-client/issues/936